src/App.vue

```diff
@@ -402,21 +478,60 @@ export default {
             }
         },
 
-        submitSectionModal() {
-            this.sections.push({
-                title: this.sectionModal.title,
-                items: [],
-            });
+        submitSectionModal(sectionKey) {
+            if (sectionKey !== undefined) {
+                this.sections[sectionKey].title = this.sectionModal.title;
+            } else {
+                this.sections.push({
+                    title: this.sectionModal.title,
+                    items: [],
+                });
+            }
 
             this.closeSectionModal();
         },
 
+        removeSectionModel(sectionKey) {
+            this.sections.splice(sectionKey, 1);
+        },
+
         closeSectionModal() {
-            this.sectionModal.isOpened = false;
+            this.$modal.hide('sectionModal');
+
             this.sectionModal.sectionKey = null;
             this.sectionModal.title = '';
         },
 
+        importClicked () {
+            this.isImport = true;
+            this.openImportExport();
+        },
+
+        openImportExport () {
```

Comment:
separate this function into `import and export` and populate the values on modal separated